content/2_Bronze/Conclusion.problems.json

```diff
@@ -31,11 +31,11 @@
       "name": "Digit Queries",
       "url": "https://cses.fi/problemset/task/2431/",
       "source": "CSES",
-      "difficulty": "Insane",
+      "difficulty": "Very Hard",
```

Comment:
Keep in mind that this is bronze though

Reply:
True, tbh I think this might be better as a silver problem. If not then insane might be fitting for bronze